### PR TITLE
Fixed multiple popup issue for delete user task

### PR DESCRIPTION
**Pull request by gnehapk**
Bugzilla: 1610668
tendrl-bug-id: https://github.com/Tendrl/ui/issues/1076

### CODE REVIEW

File: src/modules/users/user-list/users.js [hunk 1]
```diff
@@ -65,34 +64,35 @@
...
             }
 
         }];
 
-        function deleteUser(username) {
-            vm.showDeleteUserModal = true;
-            vm.deleteUser = {};
-            vm.deleteUser.username = username;
+        function deleteUser(user) {
+
```
Comment (cloudbehl) [inline src/modules/users/user-list/users.js:88]: extra space.

**Review by cloudbehl — APPROVED** [review]
LGTM

File: src/modules/users/user-list/users.js (hunk 1 above)
Comment (gnehapk) [reply src/modules/users/user-list/users.js:88]: @cloudbehl can you point me about which extra space are you referring here?

Comment (cloudbehl) [reply src/modules/users/user-list/users.js:88]: https://github.com/Tendrl/ui/pull/1077/files#diff-f6d02eab6078fb37eb753f68ead47071R88

Comment (gnehapk) [reply src/modules/users/user-list/users.js:88]: @cloudbehl That I have added intentionally for better readability.

Comment (cloudbehl) [reply src/modules/users/user-list/users.js:88]: We were not using the same aesthetic in the other functions. I think we can remove it but I am fine anyway.

**Review by a2batic — APPROVED** [review]

**Pull request merged**